File: Basics of C Programming/quadratic_equation.c
//ax^2+bx+c
#include <stdio.h>
#include <math.h>

int main(){
    int a,b,c;
    printf("Enter the coefficient of a:");
    scanf("%d",&a);
    printf("Enter the coefficient of c:");
    scanf("%d",&c);
    printf("Enter the coefficient of b:");
    scanf("%d",&b);
    double d, e;
    double dis=(b*b)-(4*a*c);
    if (dis>=0){
    d=(-b-sqrt(dis))/(2*a);
    e=(-b+sqrt(dis))/(2*a);
    printf("The two values of the quadratic equation is %lf and %lf\n",d,e);
    }else{
        printf("There are no real roots");
    }
    return 0;
}

File: python/python assignment 10 qus/quadratic equation.py
a=int(input("Enter the coefficients of a:"))
b=int(input("Enter the coefficients of b:"))
c=int(input("Enter the coefficients of c:"))
solve1=b+((((b**2)-(4*a*c))**0.5)/2*a)
solve2=b-((((b**2)-(4*a*c))**0.5)/2*a)
if type(solve1)==int or type(solve1)==float:
    if type(solve2)==int or type(solve2)==float:
        print("The solution of the quadratic equation is are,",solve1,"and",solve2)
    else:
        print("The one of the solution is a complex number")
        print("The solution is,",solve1)
else:
    print("There are no real roots")